ddtrace/appsec/_iast/_taint_tracking/CMakeLists.txt

```diff
@@ -1,16 +1,13 @@
-cmake_minimum_required(VERSION 3.19)
-include(FetchContent)
+cmake_minimum_required(VERSION 3.24)
 
 set(APP_NAME _native)
-option(BUILD_MACOS "Build for MacOS" OFF)
-option(NATIVE_TESTING "Load test subdirectories and targets" ON)
-
 project(${APP_NAME})
-
 set(CMAKE_CXX_STANDARD 17)
 
-# -U_FORTIFY_SOURCE to fix a bug in alpine and pybind11 https://github.com/pybind/pybind11/issues/1650
-# https://gitlab.alpinelinux.org/alpine/aports/-/issues/8626
```

Comment:
Why do you remove the link to the issue? I think is helpful to understand why we use the U_FORTIFY_SOURCE